core/ir/x86/decode_table.c

```diff
@@ -9290,23 +9326,38 @@ const instr_info_t evex_Wb_extensions[][4] = {
     {OP_vsqrtsd, 0xf20f5140, "vsqrtsd", Vdq, xx, KE1b, Hdq, Wsd, mrm|evex|ttt1s, x, tevexwb[266][3]},
     {OP_vsqrtsd, 0xf20f5150, "vsqrtsd", Vdq, xx, KE1b, Hdq, Usd, mrm|evex|er|ttt1s, x, END_LIST},
   }, { /* evex_W_ext 267 */
-    {OP_vpdpbusd, 0x66385008, "vpdpbusd", Ve, xx, KEd, He, We, mrm|evex|ttfv, x, END_LIST},
-    {OP_vpdpbusd, 0x66385008, "vpdpbusd", Ve, xx, KEd, He, We, mrm|evex|ttfv, x, END_LIST},
+    {OP_vpdpbusd, 0x66385008, "vpdpbusd", Ve, xx, KEd, He, We, mrm|evex|ttfv, x, tevexwb[267][1]},
+    {OP_vpdpbusd, 0x66385018, "vpdpbusd", Ve, xx, KEd, He, TYPE_W, OPSZ_4, mrm|evex|ttfv, x, END_LIST},
```

Comment:
The broadcast operand is type M, not type W, so this should be Md.

It would be nice if there were a test for the broadcast form of vpdpbusd

Reply:
Done. Added tests for vpdpbusd/vpdpbusds/vpdpwssd/vpdpwssds also.